packages/platform-sdk-btc-indexer/src/tx-parsing-helpers.ts

```diff
@@ -1,33 +1,33 @@
 import { BigNumber } from "@arkecosystem/utils";
 
-import { VIn, VOut } from "./types";
+import { Input, Output } from "./types";
 
 export const getAmount = (transaction): BigNumber =>
-	transaction.vout.reduce((c: BigNumber, v) => c.plus(Math.round(v.value * 1e8)), BigNumber.ZERO);
+	transaction.vout.reduce((carry: BigNumber, item) => carry.plus(Math.round(item.value * 1e8)), BigNumber.ZERO);
```

Comment:
I am proposing this to improve readability: https://github.com/ArkEcosystem/platform-sdk/pull/1602
Let's wait for @faustbrian's opinion

Reply:
Sure. Still we can do this in a separate PR. There's still another instance of the same further down the same file.

Reply:
Using the `BigNumber#sum` method will add extra processing because we need to call `Math.round(item.value * 1e8)` either before or after instead of directly during the `reduce`.

Reply:
```suggestion
	transaction.vout.reduce((accumulator: BigNumber, ({ value })) => accumulator.plus(Math.round(value * 1e8)), BigNumber.ZERO);
```